src/main/java/frc/robot/subsystems/led/BlinkinLedSubsystem.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot.subsystems.led;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.hardware.LedDriver;
+import frc.robot.hardware.LedDriver.BlinkinLedMode;
+// TODO Led parameters and RIO ports
+public class BlinkinLedSubsystem extends SubsystemBase {
+    private static final int port = 6; //PWM port 12
```

Comment:
Use the same port for each subsystem. Either by putting the constant in a public location (like Robot.java or LedDriver.java), or maybe making it a parameter to the constructor of each subsystem.